### ISSUE TITLE
Weird alignment - Fixing layout for iOS devices and more like the Figma Design

### ISSUE THREAD

**Issue by YuriCodes** [HetviSoni/code-fraggers#43]
I can work on this if you'd like, this is the end result, more on point with the Figma design:
 
<img width="1589" alt="Screen Shot 2022-10-15 at 10 41 25 AM" src="https://user-images.githubusercontent.com/80093500/195992430-865e8c1f-1ca2-4c0c-8688-55fbfb4f9001.png">

**Comment by HetviSoni** [comment]
Hey the above ss is not like the figma design.

**Comment by YuriCodes** [comment]
Yes! I realized and fixed it before making the PR, i also helped make it better on mobile. Check out the code and let me know

**Comment by HetviSoni** [comment]
Making it responsive for mobile is already assigned in #36 . So we can close this issue.

**Comment by YuriCodes** [comment]
Updated the code of the PR getting rid of the responsiveness but keeping the routing error fix and design.
